ivandocs/validation.md

```diff
@@ -0,0 +1,64 @@
+| 2 	| Confirm which monitoring mode is set for iOS. Note continuous mode allows for greater accuracy and persistence, but also more heavily drains battery life: https://placesdocs.com/places-services-by-adobe-documentation/configure-places-in-the-sdk/places-monitor-extension/places-monitor-api-reference#monitoring-mode-ios-only 	| Significant Changes or Continuous 	|
+| 3 	| If using more than a single Library of POIs, confirm that the appropriate libraries have been selected in the Places extension for Launch 	| Confirmed |
+| 4 	| Confirm that the latest versions of Mobile Core and Places / Places Monitor extensions have been bundled with the app via Gradle or CocoaPods	| Confirmed - for more information on recent updates see: https://placesdocs.com/places-services-by-adobe-documentation/release-notes 	|
+| 5 	| Confirm that the correct environments are configured for testing. Launch environment ID should match your Launch development environment 	|  Confirmed |
+| 6 	| Create GPX files for each POI that you want to test. GPX files can be used in local development environment to simulate a location entry. See links below for information on creating and using GPX files. <br>https://stackoverflow.com/questions/17292783/gpx-files-for-ios-simulator<br>https://mapstogpx.com/mobiledev.php<br>https://blackpixel.com/writing/2013/05/simulating-locations-with-xcode.html<br>https://qacumtester.wordpress.com/2014/02/27/location-testing-in-mobile-apps/ 	| GPX files created and loaded into app project 	|
+| 7 	| Without doing anything else, you should be able to launch the application from Android Studio or XCode and see the appropriate alert to request access for tracking location. Grant "Always Allow" permission.<br>*Recommend using a real device connected to your computer over using device simulator.* 	| Location request prompt should display on application loaded through IDE 	|
+| 8 	| Once Location permission has been accepted you should see in the application console messages that indicate ```"ACPExtensionEventName" : "requestgetnearbyplaces"```, has been called. Switching between different locations in XCode or Android studio should produce entry events for specific POIs  	| "ACPExtensionEventName" : "requestgetnearbyplaces" should be displayed as you simulate different locations.	|
+| 9 	| If the Location you have selected is close to nearby POIs the Monitor extension will start monitoring the 20 closest POIs from the current location. Message in the console will be similar to:```[AdobeExperienceSDK DEBUG <com.adobe.placesMonitor>]: Received a new list of POIs from Places:```| Switching between different locations in XCode or Android studio should produce entry events for specific POIs 	|
```

Comment:
POINT 9

**Description**
Switching between different locations in XCode or Android studio should produce entry events for specific POIs. The below logs are expected on entry to a POI

**Android**
```
PlacesExtension -  Dispatching Places Region Event for <poi name> with eventType entry
```
**iOS** 
```
[AdobeExperienceSDK TRACE <Places>]: Dispatching Places region entry event for place ID <poiId>
```